### DISCUSSION TITLE
Repository list items require to be starred

### DISCUSSION THREAD

**Discussion by recleun** [community/community#41458] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The lists are a great way to organize repositories (especially if there are many), but there is something that had been kind of bothering me. To add a repository to a list, it must be starred, which doesn't seem so right to me; Starring a repository is for people who want to stay up to date with a specific repository (like making it appear in the in the following tab of the homepage), but listing a repository is for organizing purposes. For example if you have 20 repositories and want to be able to organize them into different categories or according to the main topic of the repository.

What I'm trying to say is: Listing and starring should be separate, you shouldn't have to star a repository to list it.
I would also love to hear opinions on it, to see if it's just me or some people think the same way.

**Comment by radar-blip** [comment]
It's not just you. Lots of people think you should be able to organize repositories without starring them. 
Maybe that option just isn't available at the moment.
